src/base-config/keyboard.json

```diff
@@ -329,5 +329,8 @@
     ],
     "help.support": [
         "F1"
+    ],
+    "refactor.rename": [
+        "Ctrl-R"
```

Comment:
I believe on macOS this is translated to Cmd+R which is used to reload Brackets with Extensions.